.github/workflows/release-pr.yml

```diff
@@ -9,6 +9,6 @@ on:
 
 jobs:
    release-pr:
-      uses: OliverMKing/javascript-release-workflow/.github/workflows/release-pr.yml@main
+      uses: Azure/action-release-workflows/.github/workflows/release_js_project.yaml@a705b2ab6a3ee889f2b0d925ad0bd2f9eb733ce6
       with:
          release: ${{ github.event.inputs.release }}
```

Comment:
you need to pass in the changelogPath as input not the release

Reply:
ty updated